lib/tasks/data/pg_export.rb

```diff
@@ -450,7 +443,6 @@ def self.export_team_to_sqlite_lz4_file(team_slug, path)
             TableStrategies::Media,
             TableStrategies::Project,
             TableStrategies::ProjectMedia,
-            TableStrategies::ProjectMediaProject,
```

Comment:
@melsawy please be sure that this rake task is still able to export "project_media_projects" data, even without the model.

Reply:
OK will back this link, How can I run this rake task locally and validate the output?

Reply:
`rake check:data:export_team`